pkg/varlinkapi/volumes.go

```diff
@@ -67,7 +68,8 @@ func (i *LibpodAPI) GetVolumes(call iopodman.VarlinkCall, args []string, all boo
 			Labels:     v.Labels(),
 			MountPoint: v.MountPoint(),
 			Name:       v.Name(),
-			Options:    v.Options(),
+			// TODO change types here to be correct
+			//Options:    v.Options(),
```

Comment:
Don't we need the options as well?

Reply:
Oh, oops, that one's a mistake, reverting